packages/__docs__/src/Description/index.tsx

```diff
@@ -32,11 +32,18 @@ import type { DescriptionProps } from './props'
 class Description extends Component<DescriptionProps> {
   static propTypes = propTypes
   static allowedProps = allowedProps
+  private compiledMarkdown: any[] | null = null
+
+  componentDidMount() {
```

Comment:
shouldt this be in `componentDidUpdate`? If the props change then it wont be updated.

Reply:
Unfortunately the content of the compiledMarkdown can not be stored as state, so it cant be rewritten with componentDidUpdate. 
Luckily the props can never change, so for the time being, this solution should work fine